pkg/sypgp/sypgp.go

```diff
@@ -569,9 +569,36 @@ func SelectPrivKey(el openpgp.EntityList) (*openpgp.Entity, error) {
 	return el[n], nil
 }
 
-// SearchPubkey connects to a key server and searches for a specific key
-func SearchPubkey(search, keyserverURI, authToken string) error {
+// formatMROutput will take a machine readable input, and convert it to fit
+// on a 80x24 terminal.
+func formatMROutput(mrString string) string {
+	retList := "KEY ID    BITS  NAME/EMAIL\n\n"
```

Comment:
I would recommend looking at this package to format the output on the terminal. This will ensure proper tab separation of the rows and (imo) make the code below easier to understand since you declare a formatting string at initialization so a reader can quickly see how the line should look: https://golang.org/pkg/text/tabwriter/. An example of its use is in https://github.com/sylabs/singularity/blob/master/internal/app/singularity/remote_list.go

Reply:
Sure, I'll look into that. But, since `KEY ID`, and `BITS` are a fixed 8, and 4 characters long, maybe thats not necessary?...

Reply:
Doesn't look like its truly needed here to fix a problem. Just a code style/organization suggestion :+1: